docs/Agent/user-guide/man_page.md

```diff
@@ -1 +1,439 @@
-# Man Page
+# Man pages
+
+---
+
+### pbench-clear-results
+
+---
+
+**NAME**  
+pbench-clear-results, Clears the result directory
+
+**SYNOPSIS**  
+`pbench-clear-results [-C][--help]`
+
+**DESCRIPTION**  
+This command clears the results directory from /var/lib/pbench-agent directory
+
+**OPTIONS**  
+ -C, --config PATH  
+Path to a pbench-agent configuration file (defaults to the '\_PBENCH_AGENT_CONFIG' environment variable, if defined) [required]
+
+--help  
+Show this message and exit.
+
+---
+
+### pbench-clear-tools
+
+---
+
+**NAME**  
+pbench-clear-tools, Clear registered tools by name or group
+
+**SYNOPSIS**  
+`pbench-clear-tools [-C][-n][-g][-r][--help]
+`
+
+**DESCRIPTION**  
+Clear all tools which are registered and can filter by name of the group
+
+**OPTIONS**  
+-C, --config PATH  
+Path to a pbench-agent configuration file (defaults to the '\_PBENCH_AGENT_CONFIG' environment variable, if defined) [required]
+
+-n, --name,--names STR  
+Clear only the <tool-name> tool.
```

Comment:
The token, `<tool-name>` doesn't appear in the synopsis or in line 45, so it's not really clear what it means.  (Also, it should probably be in monospace font.)

(And, there should be a blank after the second comma...here and elsewhere.)

A similar comment applies to the `-g` and `-r` options.